papermill/iorw.py

```diff
@@ -26,10 +28,14 @@ def __init__(self):
         self.reset()
 
     def read(self, path):
+        if not fnmatch.fnmatch(os.path.basename(path), "*.*"):
```

Comment:
That's a good way to do this!

Could we also add a match warning for not `.ipynb`/`.json` as an else condition to this check? That would help when a user typos `ipynb` accidentally.

Reply:
yes made this change

Reply:
facing a new issue over here. 

$ mv .coverage $TRAVIS_BUILD_DIR/.coverage
mv: cannot stat ‘.coverage’: No such file or directory
The command "mv .coverage $TRAVIS_BUILD_DIR/.coverage" exited with 1.